runatlantis.io/docs/apply-requirements.md

```diff
@@ -66,6 +66,7 @@ If you set up Protected Branches then you can enforce:
 * Requiring certain people to have reviewed and approved the pull request
 * Requiring `CODEOWNERS` to have reviewed and approved the pull request
 * Requiring that the branch is up to date with `master`
+* Requiring that the Atlantis user has write/push permissions to the base branch of the pull request
```

Comment:
@lkysow does this requirement make sense? Obviously it makes sense if you want auto merging but that has not been implemented. If you don't want auto merging why does it need push to the base branch? It should really only need read/pull I would think though I admit I have not been keeping up with a lot of the awesome stuff coming out.

Reply:
Hrm, maybe I should rephrase this, AFAIK it's only needed when this branch protection option is checked:
![image](https://user-images.githubusercontent.com/12276488/54193547-9a9b8500-44ba-11e9-8235-ee9a5f4e0764.png)

If this is selected, and Atlantis isn't in the list of users/groups, then it won't consider a PR mergeable.

Reply:
I will see if I can replicate that tomorrow with the latest released version as I am several versions behind. We are using branch protection and atlantis github user has not been given permission to merge to protected branches.

Reply:
OK, removed.